.syncpackrc.js

```diff
@@ -0,0 +1,38 @@
+module.exports = {
+  versionGroups: [
+    {
+      label: 'Internal packages that have inconsistent version usage',
+      packages: ['**'],
+      dependencies: [
+        '@kadena/chainweb-node-client',
+        '@kadena/client',
+        '@kadena/fonts',
+        '@kadena/pactjs-cli',
+        '@kadena-dev/eslint-plugin',
+      ],
+      isIgnored: true, // Toggle flag or or remove group to see inconsistencies
+    },
+    {
+      label: 'Internal dev packages are pinned to `workspace:*`',
+      packages: ['**'],
+      dependencies: ['@kadena-dev/*', '@kadena/types'],
+      dependencyTypes: ['dev'],
+      pinVersion: 'workspace:*',
+    },
+    {
+      label: 'Internal production packages are pinned to `workspace:*`',
+      packages: ['**'],
+      dependencies: ['kadena.js', '@kadena/*'],
+      dependencyTypes: ['prod'],
+      pinVersion: 'workspace:*',
+    },
+    {
+      label:
+        'Types and internal dev packages are banned from dependencies (only allowed in devDependencies)',
+      packages: ['**'],
+      dependencies: ['@types/*', '*/types', '@kadena-dev/*'],
+      dependencyTypes: ['prod'],
+      isBanned: true,
```

Comment:
[syncpack](https://github.com/JamieMason/syncpack) requires a bit of configuration, but is actually pretty powerful and has some advantages over the Rush solution:

- It also considers internal packages
- We can properly separate production and dev packages
- We can ban all sorts of packages containing types from `dependencies`

This PR contains quite some fixes detected by syncpack.